src/client/src/components/DraggableSidebarItem/index.tsx

```diff
@@ -43,6 +49,8 @@ const DraggableSidebarItem = ({
 }: {
   page?: ISelected;
   text?: string;
+  azureFunctionsSvg?: boolean;
```

Comment:
NIT: since this is a boolean, can we name it isAzureFunc... or hasAzureFunc...
Same for cosmosDBSvg

Reply:
yup! I just changed it